inicheck/checkers.py

```diff
@@ -127,11 +128,11 @@ def check(self):
         Returns:
             msgs: None if the entry is valid, else returns self.message
         """
-        valids = []
+
         issues = []
 
         for v in mk_lst(self.values):
-            valid, issue = self.is_valid(v)
+            valid, issue = self.is_valid(v)  # noqa
```

Comment:
Curious what warning was raised by Flake8?

Reply:
The `valid` output wasn't being used so it was complaining.

Reply:
You can mark unused variables with an `_` and Flake8 will not complain.
```python
_value, issue = self.is_valid(v)
```

Reply:
Acutally turns out that the lint problem is because in this particular class `GenericCheck` is meant to be the base class which other checkers inherit from. The `is_valid()` method is a place holder in `GenericCheck` that doesn't return any values.